js/stat.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+var CLOUD_WIDTH = 420;
+var CLOUD_HEIGHT = 270;
+var CLOUD_X = 100;
+var CLOUD_Y = 10;
+var GAP = 10;
+var CLOUD_GAP = 20;
+var FONT_GAP = 20;
+var TEXT_HEIGHT = 16;
+var BAR_WIDTH = 40;
+var BAR_GAP = 50;
+var MAX_BAR_HEIGHT = 150;
+
+
+var renderCloud = function (ctx, x, y, color) {
+  ctx.fillStyle = color;
+  ctx.fillRect(x, y, CLOUD_WIDTH, CLOUD_HEIGHT);
+};
+
+var getMaxElement = function (arr) {
+  var maxElement = arr[0];
+  for (var i = 1; i < arr.length; i += 1) {
```

Comment:
```suggestion
  for (var i = 0; i < arr.length; i++) {
```

Особенность джава скрипта, что все итерации начинаются с нуля, первый элемент всегда индекс 0. (Январь, например, не 1й (первый) месяц, а 0й (нулевой) месяц).

Так же, несмотря на то, что с помощью `i += 1` мы тоже увеличиваем итератор на единицу, есть более компактная запись: `i++` (почитай про оператор инкримента), и также, эта компактная запись общепринята в таких циклах.